code/modules/gear_presets/clf.dm

```diff
@@ -574,11 +550,129 @@
 
 //*****************************************************************************************************/
 
-/datum/equipment_preset/clf/specialist/cryo
-	name = "CLF Cryo Specialist"
+/datum/equipment_preset/clf/commander
+	name = "CLF Cell Commander"
+	flags = EQUIPMENT_PRESET_EXTRA
+	assignment = JOB_CLF_COMMANDER
+	rank = JOB_CLF_COMMANDER
+	role_comm_title = "CMDR"
+	skills = /datum/skills/clf/commander
+
+/datum/equipment_preset/clf/commander/load_gear(mob/living/carbon/human/H)
```

Comment:
You should update the single var usage here (H) to use a more descriptive name.

See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#use-descriptive-and-obvious-names

Reply:
The re-addition of that text was mainly cus I screwed up deleting the CLF cryo list, so that is legacy code. I shall, however, oblige in making this area of code better.